docs/public/using/backend/rpcapi-reference.md

```diff
@@ -1313,29 +1316,9 @@ A [*batch id*][Batch id].
 
 #### `"error"`
 
-* You cannot create a new batch job if a *batch* with unfinished *tests* already
-  exists for this [*username*][Username].
 * If the given `profile` is not among the [available profiles][Profile sections],
   a user error is returned, see the [profile name section][profile name].
```

Comment:
The bullet point list has only one item left, so how about turning it into a regular paragraph?
```suggestion
If the given `profile` is not among the [available profiles][Profile sections], a
user error is returned, see the [profile name section][profile name].
```